package.json

```diff
@@ -96,6 +96,8 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "^0.10.0",
+    "@angular-devkit/core": "^7.3.3",
```

Comment:
Revert these pinned dependencies. They are installed with the `@angular/cli` dependencies.

Reply:
removed the import.